src/main/java/org/marketplace/configuration/SecurityConfig.java

```diff
@@ -53,11 +59,11 @@ public AuthenticationManager authenticationManager(AuthenticationConfiguration a
         return authenticationConfiguration.getAuthenticationManager();
     }
 
-//    @Autowired
-//    public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
-//        auth.inMemoryAuthentication()
-//                .withUser("defaultUser")
-//                .password(passwordEncoder().encode("defaultPass"))
-//                .roles("USER");
-//    }
+    @Autowired
+    public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
+        auth.inMemoryAuthentication()
+                .withUser("defaultUser")
+                .password(passwordEncoder().encode("defaultPass"))
+                .roles("USER");
+    }
```

Comment:
i think there might be no need to keep in memory user as we already had added default user here
```
org/marketplace/configuration/DataLoader.java:26
```